### ISSUE TITLE
Rename the repository?

### ISSUE THREAD

**Issue by josephworks** [OxideWaveLength/Minecraft-Hack-BaseClient#58]
Minecraft-Hack-BaseClient doesn't sound like the best name for the project, and I think we should switch to a more simple name like BaseClient, HackedMCClient, ClientBase, MCClient, HackedClient, BaseHacks, or another short name.

**Comment by josephworks** [comment]
Also, TBH I think that the username OxideWaveLength sounds pretty cool, but the repo name is a real killer.

**Comment by OxideWaveLength** [comment]
Yes, I agree with you, I will be searching for a better name, and of course suggestions are welcome

**Comment by lukeacat** [comment]
The name should be MCBase and you should add tags so it's easier to find

**Comment by AverageComet250** [comment]
I think adding tags is a good idea but I think BaseMC sounds better than MCBase. Of course, the final decision is upto OxideWaveLength.

I think the idea of this being a client that anyone can add to is something I like, so the name saying that it's a 'base' is near necessary.

**Comment by josephworks** [comment]
BaseMC sounds like a flashy server name imo. This is GitHub, the name simply needs to represent what the project is. I would use the word client in there, no need to use the word mc or Minecraft, and using base is acceptable. ClientBase for example perfectly represents what this project is.

**Comment by AverageComet250** [comment]
I see what you mean. However, there are millions of games that have modded clients, so having Minecraft or MC in the name makes it easier for people to know it's a Minecraft hack client and not a Fortnite hack client.

**Comment by josephworks** [comment]
Look at Paper, or glowstone, it essentials x and reconsider. You should use
minecraft as a tag, not part of repo name.

**Comment by OxideWaveLength** [comment]
Shall the repository be renamed to ClientBase then?

**Comment by AverageComet250** [comment]
Yeah. It's a good name. It perfectly describes the client but to counter what @josephworks wrote, what about Forge. It's full name is Minecraft Forge. What about MultiMC. Glowstone makes you think of Minecraft cause it's an item in Minecraft. And paper's full name is PaperMC. I can't think of a way to counter essentials X though.

**Comment by josephworks** [comment]
Full names are different from repository / project names although short repository names can be used as full names.

**Comment by AverageComet250** [comment]
In that case, I think the _Repository_ should be named **ClientBase** and the _actual client/project_ should be named **ClientBaseMC**

**Comment by neroni4** [comment]
Wave, a can contact with you ?

**Comment by OxideWaveLength** [comment]
@neroni4 you can join the client's discord server through this invite link: https://discord.gg/pDweRGz or open a new issue.

**Comment by josephworks** [comment]
@OxideWaveLength what are your thoughts on the repo name?

**Comment by OxideWaveLength** [comment]
> In that case, I think the _Repository_ should be named **ClientBase** and the _actual client/project_ should be named **ClientBaseMC**

I think I'm going for this. Will rename the repository to "ClientBase" and the client to "ClientBaseMC"

**Comment by josephworks** [comment]
Hmm, I’m thinking vice-versa. Repo name ClientBaseMC, package name net.oxidewavelength.clientbase.
